tdrs-backend/tdpservice/scheduling/tasks.py

```diff
@@ -41,3 +44,34 @@ def users_to_deactivate(days):
         last_login__gte=datetime.now(tz=timezone.utc) - timedelta(days=days+1),
         account_approval_status=AccountApprovalStatusChoices.APPROVED,
         )
+
+def get_ofa_admin_user_emails():
+    """Return a list of OFA System Admin users."""
+    return [user.email for user in User.objects.filter(
+        groups='4'
```

Comment:
i prefer `groups=Group.objects.get(name='whatever')` for the clarity, though what you have is probably faster